tests/apollo/util/skvbc.py

```diff
@@ -176,18 +177,65 @@ async def send_indefinite_write_requests(self, client=None, delay=.1):
                     pass
                 await trio.sleep(delay)
 
-    async def write_known_kv(self):
+    async def write_known_kv(self, kv=None, client=None, long_exec=False):
         with log.start_action(action_type="write_known_kv"):
-            client = self.bft_network.random_client()
-
-            key = self.random_key()
-            val = self.random_value()
-            reply = await client.write(
-                self.write_req([], [(key, val)], 0))
-            reply = self.parse_reply(reply)
-            assert reply.success
+            if client == None:
+                client = self.bft_network.random_client()
+            if kv == None:
+                kv_input = False
+                key = self.random_key()
+                val = self.random_value()
+                kv = [(key, val)]
+            else:
+                kv_input = True
+            read_version = 0
+            if self.tracker != None:
```

Comment:
`if self.tracker is not None` sounds more like Python. Same comment throughout.

Reply:
Agreed it's better style. I'll change it.